app/Http/Controllers/CollectionController.php

```diff
@@ -144,6 +145,10 @@ public function view(Request $request, Collection $collection): Response
         $reportAvailableIn = RateLimiterHelpers::collectionReportAvailableInHumanReadable($request, $collection);
 
         if (! $collection->recentlyViewed()) {
+            if ($collection->banner() === null) {
```

Comment:
here I'd add a check of a week before refetching. Otherwise this change will cause the banners to not update at all if they existed prior, which means that any changes to banners would not be picked up

Reply:
yeah would need something specific for the banner to avoid it being skipped when other aspects of the model get updated